Doc/library/subprocess.rst

```diff
@@ -466,9 +466,9 @@ functions.
       The *pass_fds* parameter was added.
 
    If *cwd* is not ``None``, the function changes the working directory to
-   *cwd* before executing the child.  In particular, the function looks for
-   *executable* (or for the first item in *args*) relative to *cwd* if the
-   executable path is a relative path.
+   *cwd* before executing the child. *cwd* also accepts a `os.PathLike` object.
```

Comment:
It would be better to follow the style in the rest of stdlib:

```rst
[...] child.  *cmd* can be a :class:`str` and :term:`path-like <path-like object>` object.
```